src/cljs/main/broadfcui/common.cljs

```diff
@@ -1,10 +1,10 @@
-(ns org.broadinstitute.firecloud-ui.common
+(ns broadfcui.common
```

Comment:
All of this file moving could cause issues for people when merging previously created branches. I think it would be a good idea to warn people before merging this to develop.